ADBench/run-all.ps1

```diff
@@ -329,21 +350,21 @@ Class Tool {
     }
 
     # Run a single test
-    [void] run ([string]$objective, [string]$dir_in, [string]$dir_out, [string]$fn) {
-        if ($objective.contains("Eigen")) { $out_name = "$($this.name)_Eigen" }
-        elseif ($objective.endswith("SPLIT")) { $out_name = "$($this.name)_split" }
-        else { $out_name = $this.name }
-        $output_file = "${dir_out}${fn}_times_${out_name}.txt"
+    # Returns true if there was a timeout and false otherwise
+    [bool] run ([string]$objective, [string]$dir_in, [string]$dir_out, [string]$fn) {
```

Comment:
Usually, returned `true` value indicates success, and `false` some sort of error. That being inversed may be confusing, especially since the function is called `run` and not something like `did_timeout`.

Reply:
Maybe, define a RunResult enumeration with values like `Success`, `Timeout`, `Skipped`, `ResultIncorrect`, etc. and return it?

Reply:
Yes, I agree with this. Changes are performed